README.md
Source files for robjhyndman.com
=================================

All content and theme files for https://robjhyndman.com, using [Blogdown](https://github.com/rstudio/blogdown) and [Hugo](http://gohugo.io/) to generate the site. 

The design is inspired by http://kieranhealy.org and is a modification of [hugo-finite](https://github.com/lambdafu/hugo-finite). 


### Licenses of third-party components

* kube: MIT License
* hugo-finite: MIT License
* hugo-academic: MIT License
* Highlight.js: MIT License
* MathJax: Apache License 2.0
* Foundation 6.2.3 by Zurb: MIT License
* Font Awesome 4.6.3: SIL OFL 1.1 (font), MIT License (CSS)
* Academicons 1.7.0: SIL OFL 1.1 (font), MIT License (CSS)
* Google fonts: OFL (SIL Open Font License)



content/news.md
---
title: "In the news"
slug: "in-the-news"
---

  * ["Monash University’s Rob Hyndman awarded the Pitman Medal 2021"](https://busycontinent.com/monash-universitys-rob-hyndman-awarded/), Busy Continent, 28 July 2021.
  * ["Top Stats Society Honour for ACEMS Chief Investigator"](https://acems.org.au/news/rob-hyndman-pitman-medal), ACEMS, 9 July 2021.
  * ["Rob Hyndman awarded the Pitman Medal 2021"](https://statsoc.org.au/News-and-media-releases/10736720), Statistical Society of Australia, 8 July 2021.
  * ["IIF Fellow & Distinguished Researcher: Rob Hyndman"](https://forecasters.org/blog/2021/07/02/iif-fellow-distinguished-researcher-rob-hyndman/), International Institute of Forecasters, 2 July 2021.
  * ["International Journal of Forecasting Announces Best Papers from the M4 special issue"](https://forecasters.org/blog/2021/06/30/international-journal-of-forecasting-announces-best-papers-from-the-m4-special-issue/), International Institute of Forecasters, 30 June 2021.
  * ["All eyes online"](https://www.monash.edu/business/awards/george-athanasopoulos-and-rob-hyndman), "Monash Awards", 10 June 2021.
  * [“This is really a marathon.” Meet the statistician helping forecast the spread of COVID-19](https://www2.monash.edu/impact/articles/covid-19/this-is-really-a-marathon-meet-the-statistician-helping-forecast-the-spread-of-covid-19/), *Monash Impact*, 25 May 2021.
  * [The Australia Academy of Science Elects ACEMS Chief Investigator as New Fellow](https://acems.org.au/news/rob-hyndman-science-academy-fellow), *ACEMS News*, 25 May 2021.
  * [Leading Monash University scientists elected Fellows of Australian Academy of Science](https://www.monash.edu/news/articles/leading-monash-university-scientists-elected-fellow-of-australian-academy-of-science), *Monash News*, 25 May 2021.
  * [Distinguished statistician elected Fellow of the Australian Academy of Science](https://www.monash.edu/news/articles/distinguished-statistician-elected-fellow-of-the-australian-academy-of-science), *Monash News*, 25 May 2021.
  * [Twenty-two Australians recognised among our nation’s most distinguished scientists](https://www.science.org.au/news-and-events/news-and-media-releases/twenty-two-australians-recognised-among-nations-distinguished-scientists), *Australian Academy of Science*, 25 May 2021.
  * [Hundreds of academics back self-ID sex question in Scottish census](https://www.heraldscotland.com/news/19208054.hundreds-academics-back-self-id-sex-question-scottish-census/), *The Herald* (Scotland), 4 April 2021.
  * [Two ACEMS Researchers Named Fellows by Social Sciences Academy](https://acems.org.au/news/acems-social-sciences-fellows), *ACEMS News*, 16 November 2020.
  * [Cybersecurity: How data scrutiny can thwart attacks](https://www2.monash.edu/impact/articles/big-data/cybersecurity-how-data-scrutiny-can-thwart-attacks/), *Monash Impact*, 12 November 2020.
  * [Respected Business researchers honoured as Fellows of Academy of Social Sciences of Australia](https://www.monash.edu/business/news/2020/respected-business-researchers-honoured-as-fellows-of-academy-of-social-sciences-of-australia), *Monash Business News*, 11 November 2020.
  * [What's the pandemic forecast?](https://www.arc.gov.au/news-publications/media/feature-articles/whats-pandemic-forecast), *ARC feature article*, 30 September 2020.
  * [What's the pandemic forecast?](https://acems.org.au/news/covid-forecasting-team), *ACEMS news*, 12 August 2020.
  * [Inaugural Zema Energy Studies Scholarship launched](https://www.energymagazine.com.au/inaugural-zema-energy-studies-scholarship-launched/), *Energy Magazine*, 5 March 2020.
  * [New scholarship program supports Monash students to address Australia's energy sector challenges](https://www.monash.edu/news/articles/new-scholarship-program-supports-monash-students-to-address-australias-energy-sector-challenges), *Monash News*, 4 March 2020.
  * [Big data, big deal: what a career in data science really takes](https://thenewdaily.com.au/sponsored/2019/11/27/big-data-big-deal-what-a-career-in-data-science-really-takes/), *The New Daily*, 27 November 2019.
  * [How to boost your career one gigabyte at a time](https://thenewdaily.com.au/sponsored/2019/10/24/how-to-boost-your-career-one-gigabyte-at-a-time/), *The New Daily*, 24 October 2019.
  * [River and reef water quality monitoring revolutionized](https://www.technologynetworks.com/applied-sciences/news/river-and-reef-water-quality-monitoring-revolutionized-323567), *Technology Networks*, 4 September 2019.
  * [Revolutionizing water quality monitoring for our rivers and reef](https://www.sciencedaily.com/releases/2019/09/190903091446.htm), *Science News*, 3 September 2019.
  * [Revolutionizing water quality monitoring for our rivers and reef](https://www.eurekalert.org/pub_releases/2019-09/quot-rwq090219.php), *EurekAlert!*, 3 September 2019.
  * [Labor’s climate policy is second-rate – because the government’s policy is fifth-rate](https://thenewdaily.com.au/money/finance-news/2019/05/06/labor-climate-policy-costings/), *The New Daily*, 6 May 2019.
  * [Announcing the winners of the Facebook Statistics for Improving Insights and Decisions research awards](https://research.fb.com/announcing-the-winners-of-the-statistics-for-improving-insights-and-decisions-research-awards/), *Facebook Research*, 25 January 2019.
  * [Forecasting: putting the smart into smart meters](https://www2.monash.edu/impact/articles/econometrics/forecasting-putting-the-smart-into-smart-meters/), *Monash Impact*, 24 October 2018.
  * [QUT works with ABS to boost use of big data](https://www.qut.edu.au/news?news-id=133908), *QUT News*, 3 August 2018.
  * [Future predictions: revolutionising the accuracy of business forecasting](https://www2.monash.edu/impact/articles/econometrics/future-predictions-revolutionising-the-accuracy-of-business-forecasting/), *Monash Impact*, 11 April 2018.

public/Rfiles/funcfor.R
require(demography)

##########################
#### FRENCH MORTALITY ####
##########################

# Plot data
plot(fr.mort,series="male")

# Figure 1 : B&W version for paper with only a few years
plot(fr.mort,series="male",
    years=c(1899,1918,1957,1977,1997),type="p",cex=0.6,col=1,pch=c(1,15,22,2,16),
    main="France: Male death rates")
legend(5,1.5,legend=paste(c(1899,1918,1957,1977,1997)),col=1,pch=c(1,15,22,2,16),cex=0.8)

# Smoothing
france.sm <- smooth.demogdata(fr.mort,obs.var="theoretical")
i=c(1901,1973);plot(fr.mort,years=i,type="p",pch=1);lines(france.sm,years=i)

# Fit model to data up to 2001.
france.short <- extract.years(france.sm,1899:2001)
frmort.fit <- fdm(france.short,series="Male",order=4,lambda=3,max.age=110,method="M")
frmort.fcast <- forecast(frmort.fit,h=50,method="arima",level=80,adjust=FALSE)

# Figure 6: fitted basis functions and forecast coefficients
plot(frmort.fcast,"c",ylab1="B",ylab2="C",mean.lab="mux",main.title="",
    interaction.title="",shadecol=gray(0.9),fcol="black",outlier.col=1,outlier.pch=18)

# Forecasts with prediction intervals
years <- 2001+c(1,20)
plot(frmort.fcast,series="Male",years=years,ylim=c(-9.7,0),lty=1:2,col=1,max.age=109)
lines(frmort.fcast,series="upper",years=years,lty=1:2,col=1,max.age=109)
lines(frmort.fcast,series="lower",years=years,lty=1:2,col=1,max.age=109)
legend(0,0,legend=paste(years),lty=1:2,cex=0.7)

plot(frmort.fcast,series="Male",years=years,lty=1:2,col=1,max.age=100,transform=FALSE)
lines(frmort.fcast,series="upper",years=years,transform=FALSE,lty=1:2,col=1,max.age=100)
lines(frmort.fcast,series="lower",years=years,transform=FALSE,lty=1:2,col=1,max.age=100)
legend(0,0.43,legend=paste(years),lty=1:2,cex=0.7)
rm(years)


##############################
#### AUSTRALIAN FERTILITY ####
##############################

# Use data up to 2000
aus.fert$lambda <- 0 # Use logs for transformation
ausfert <- extract.years(aus.fert,1921:2000)
ausfert$rate$female <- rbind(rep(0.005,80),ausfert$rate$female,rep(0.005,80))
ausfert$pop$female <- rbind(ausfert$pop$female[1,],ausfert$pop$female,ausfert$pop$female[7,])
rownames(ausfert$rate$female) <- rownames(ausfert$pop$female) <- c("<15","15-19","20-24","25-29","30-34","35-39","40-44","45-49",">49")
ausfert$age <- c(13,17,22,27,32,37,42,47,52)

# Figure 2: plot separate time series for each age group
plot(aus.fert,plot.type="time",xlim=c(1920,2005),lty=1:7,show.labels=FALSE,transform=TRUE)
text(rep(2004,7)-c(0,0,8,8,0,0,0),log(aus.fert$rate[[1]])[,80]+c(0,.1,.4,-.2,-.1,0,0),
    rownames(aus.fert$rate$female),cex=0.7)

# Figure 2. published version
plot(aus.fert,years=1921:2000,plot.type="time",xlim=c(1920,2005),lty=1:7,show.labels=FALSE,col=1,transform=TRUE,main="",xlab="Year")
text(rep(2004,7)-c(0,0,8,8,0,0,0),log(aus.fert$rate[[1]])[,80]+c(0,.1,.4,-.2,-.1,0,0),
    rownames(aus.fert$rate$female),cex=0.7)

# Smooth data
fert.sm <- smooth.demogdata(ausfert,age.grid=13:52,obs.var="theoretical")
plot(fert.sm,transform=TRUE);points(ausfert,transform=TRUE)

# Plot of smoothed data
years=c(1921,1940,1960,1980,2000)
plot(fert.sm,years=years,col=1:5,main="")
points(ausfert,years=years,pch=1,col=1:5,cex=0.6)
legend("topright",legend=paste(years),lty=1,col=1:5,cex=0.9)

plot(fert.sm,years=years,col=1:5,main="",transform=TRUE)
points(ausfert,years=years,pch=1,col=1:5,cex=0.6,transform=TRUE)
legend("topright",legend=paste(years),lty=1,col=1:5,cex=0.9)

# Figure 3:
plot(fert.sm,years=years,lty=1:5,col=1,main="",transform=TRUE)
points(ausfert,years=years,pch=1,col=1,cex=0.6,transform=TRUE)
legend(20,-1,legend=paste(years),lty=1:5,cex=0.9)
rm(years)

# Fit model and forecasting using ARIMA
ausfert.fit <- fdm(fert.sm,order=3,method="classical")
fert.fcast <- forecast(ausfert.fit,h=20,level=80,method="arima",adjust=FALSE)

# Figure 4: Plot of forecast coefficients
plot(fert.fcast,plot.type="c",ylab2="C",ylab1="B",mean.lab="mux",main.title="",interaction.title="",shadecols=gray(0.9),fcol=1)

# Figure 5: Plot of forecast curves
years=2000+c(1,20)
plot(fert.fcast,years=years,lty=1:2,col=1,ylim=c(0,160),yaxs="i",main="")
lines(fert.fcast,series="upper",years=years,lty=1:2,col=1)
lines(fert.fcast,series="lower",years=years,lty=1:2,col=1)
legend(15,145,legend=paste(years),lty=1:2,cex=0.7)
rm(years)
